src/components/SourceFooter.css

```diff
@@ -12,50 +12,39 @@
   user-select: none;
 }
 
-html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+.source-footer__command-bar {
```

Comment:
@devtools-html/debugger-core @helenvholmes 

I believe this is the first time we'd use BEM for a component that is used in several places. We could potentially do this with the close button and sizes.

I personally am more comfortable with well scoped classes `.command-bar` `.source-footer > .command-bar`, but that's probably because of my experience set and I'd like to have an open mind on this issue.